packages/php-storage-driver-common/proto/bucket.proto

```diff
@@ -95,11 +95,12 @@ message UnshareBucketCommand {
 }
 
 message GrantBucketAccessToReadOnlyRoleCommand {
-  string bucketObjectName = 1;
```

Comment:
@zajca nevim, jestli se to dělá takhle u protobufferu... nedělá se to tak, že když to chceš změnit, tak ten původní zahodíš a přidáš ten nový na konec? (jakože by to byla 7)

Reply:
Jop :( https://protobuf.dev/programming-guides/dos-donts/#reuse-number

Reply:
Toereticky v našem use-case je to fuk, protože když teď smažeš 1 tak ti zmizíja metody a je to BC, ale bylo by dobré sa držat toho, že sa fieldy nepřejmenovávajů ani nerejůsujů čísla. Ono je to enom číslo, musíš sa odpoutat od pozemského myšlení, že to má jít hezky za sebů, je to prostě šumák.

Reply:
No, ale to číslo je interní identifikátor, ne? 
Takže teoreticky může přijít message, která ho ještě má, ale čeká tam string. A tím pádem to vyhučí na deserializaci a nezjistíš, že to byla stará message protože to bude jen garbage, ne?

Reply:
To by mělo v tom protobuf projít, ale nezkůšal sem to. interně to použije podobným způsobem jakkdyž uděláš serialize objektu v PHP, ale mělo by to byt reálně proti tomuto odolné.

Reply:
kazdopadne sa drzim best practice google a necham to takto